drafts/use-cases/1.1.security-considerations.md

```diff
@@ -10,23 +10,30 @@ There are several possibilities here. One of them would be to use a *@graph* not
 
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@graph": [
         {
             "@id": "/api",
-            "operation": [
-                {
-                    "@type": "hydra:Operation",
-                    "title": "List events",
-                    "method": "GET",
-                    "returns": "hydra:Collection"
-                },
-                {
-                    "@type": "hydra:Operation",
-                    "title": "Create new event",
-                    "method": "POST",
-                    "expects": "schema:Event"
-                }
-            ]
+            "@type": "hydra:EntryPoint",
+            "#events": {
```

Comment:
I'd like to leave `#events` in this PR and I already plan another PR which would propose alternative approach to `hydra:Link` and rely just on `schema:Event` used together with [`void:classPartition`](http://vocab.deri.ie/void#classPartition). Which would remove that `#events` all together since IMO it doesn't really address need to discover relevant collection in the dataset exposed by this API.